notebooks/official/custom/sdk-custom-image-classification-online.ipynb

```diff
@@ -142,342 +142,183 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    MODEL_DISPLAY_NAME = "cifar10-" + "unique"
</pre><p>Just use a string, no need to concatenate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
done